benchconnect/tests/test_augment.py

```diff
@@ -0,0 +1,68 @@
+import json
+import shutil
+from typing import Callable
+
+import pytest
+from benchadapt.result import BenchmarkResult
+from benchadapt.run import BenchmarkRun
+from click.testing import CliRunner
+
+from benchconnect._augment import augment_blob, result, run
+
+minimal_result = {"stats": {"data": [1, 2], "unit": "s"}}
+minimal_run = {}
+
+
+@pytest.mark.parametrize(
+    "params", [(minimal_result, BenchmarkResult), (minimal_run, BenchmarkRun)]
+)
+def test_augment_blob(params):
+    raw_blob, cls = params
+    augmented_blob = augment_blob(raw_blob, cls=cls)
+    assert augmented_blob.keys() > raw_blob.keys()
```

Comment:
nit: parametrize allows multiple inputs, so you could do this to simplify slightly
```suggestion
@pytest.mark.parametrize(
    ["raw_blob", "cls"], [(minimal_result, BenchmarkResult), (minimal_run, BenchmarkRun)]
)
def test_augment_blob(raw_blob, cls):
    augmented_blob = augment_blob(raw_blob, cls=cls)
    assert augmented_blob.keys() > raw_blob.keys()
```

Reply:
Interesting! But I find it less than obvious that it uses the first list to unpack cases—at a glance I'd guess each of the elements of the latter list are args that will get crossed like if you had >1 `@pytest.mark.parameterize` calls. But it looks like it accepts a tuple for the first arg, which makes it read a little more clearly to me

Reply:
oh yeah it accepts tuples too, or this (IMO gross) format:
```py
@pytest.mark.parametrize(
    "raw_blob,cls", [(minimal_result, BenchmarkResult), (minimal_run, BenchmarkRun)]
)
```